pkg/server/config/manager.go

```diff
@@ -64,6 +65,32 @@ func NewManager(config MultusConf) (*Manager, error) {
 	return newManager(config, defaultPluginName)
 }
 
+// atomicFileWrite writes our configs atomically to avoid partial reads
+func atomicFileWrite(filename string, data []byte, perm os.FileMode) error {
+	// Create a temporary file in the same directory as the destination file
+	dir := filepath.Dir(filename)
+	tmpFile, err := ioutil.TempFile(dir, ".multus.tmp")
```

Comment:
ioutil.TempFile() is deprecated as https://pkg.go.dev/io/ioutil#TempFile so please change that.

Reply:
https://pkg.go.dev/os#CreateTemp